pandas/core/apply.py

```diff
@@ -542,12 +527,13 @@ def maybe_apply_multiple(self) -> Optional[FrameOrSeriesUnion]:
 
     def validate_dictlike_arg(
         self, how: str, obj: FrameOrSeriesUnion, func: AggFuncTypeDict
-    ) -> None:
+    ) -> AggFuncTypeDict:
         """
-        Raise if dict-like argument is invalid.
+        Handler for dict-like argument.
 
         Ensures that necessary columns exist if obj is a DataFrame, and
-        that a nested renamer is not passed.
+        that a nested renamer is not passed. Also normalizes to all lists
```

Comment:
maybe not here if too many changes for a backport, but maybe change function name to say `normalize_dictlike_arg`

Reply:
@simonjayhawkins Thanks for the suggestion. That's actually how I started this out, but then changed it to validate recalling `util._validators` (some of those do only validate, but a bunch also normalize). I'm seeing a mix of normalize or validate used. I'll change to normalize here, but maybe we should make this consistent.

Related: #19171